src/components/about/homePage.tsx

```diff
@@ -80,8 +80,13 @@ const HomePage = () => {
           >
             <div className="font-seoul p flex flex-col justify-center space-y-6 pt-8 pb-8 text-center">
               <p>Find us on the Highlander Link:</p>
-              <a href="https://highlanderlink.ucr.edu/organization/rbrains">
-                https://highlanderlink.ucr.edu/organization/rbrains
+              <a
```

Comment:
Use next/link: put this at the top import Link from "next/link"; to import link then change the <a> to <Link> as this will allow for client side navigation